src/elm/Page/Dashboard/Analysis.elm

```diff
@@ -200,7 +280,7 @@ viewClaim ({ shared, accountName, selectedCommunity } as loggedIn) f claim =
                     [ class "button button-secondary w-full font-medium mb-2"
                     , Route.href <| Route.Claim selectedCommunity claim.action.objective.id claim.action.id claim.id
                     ]
-                    [ text_ "dashboard.all_analysis.more_details" ]
+                    [ text_ "all_analysis.more_details" ]
                 ]
 
           else
```

Comment:
I agree. Any thoughts or reasons for this @heltonlr?

Reply:
These actions should appear in the filter under "under review", correct. I did not show the status as it would be clear that the activity was under review, as they are asking for my assessment. But ok, we can also show the status.

Reply:
ok, thanks @heltonlr and @amiskov! Added!